app-network/rustdesk-server/autobuild/overrides/usr/lib/systemd/system/rustdesk-hbbs.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=RustDesk Server - HBBS (Signal Server)
+After=network.target
+
+[Service]
+User=rustdesk-server
+Group=rustdesk-server
```

Comment:
Was this specified by the upstream? If not, can we try `rustdesk` so we save a dash?